cloudbuild.yaml

```
@@ -53,9 +53,29 @@ steps:
         ".",
       ]
 
-  - name: "us.gcr.io/$PROJECT_ID/relay:$COMMIT_SHA"
-    args: ['--help']
-    id: "smoke test"
+  # On-premise Integration tests
+  - name: 'gcr.io/$PROJECT_ID/docker-compose'
+    entrypoint: 'bash'
+    env:
+    - 'RELAY_IMAGE=us.gcr.io/$PROJECT_ID/relay:$COMMIT_SHA'
+    - 'SENTRY_TEST_HOST=http://nginx'
+    - 'CI=1'
+    args:
+    - '-e'
+    - '-c'
+    - |
+      mkdir onpremise && cd onpremise
+      # TODO(byk): We may also build this part as a builder image everytime
+      #            there's a push to onpremise and use that image for
+      curl -L "https://github.com/getsentry/onpremise/archive/byk/feat/custom-images.tar.gz" | tar xzf - --strip-components=1
```

This will become `master.tar.gz` when getsentry/onpremise#602 is merged.
```suggestion
      curl -L "https://github.com/getsentry/onpremise/archive/master.tar.gz" | tar xzf - --strip-components=1
```